.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  env: {
+    browser: true,
+    commonjs: true,
+    es6: true,
+    node: true,
+  },
+  extends: 'eslint:recommended',
+  parserOptions: {
+    sourceType: 'module',
+  },
+  rules: {
+    'comma-dangle': ['error', 'always-multiline'],
+    indent: ['error', 2],
+    'linebreak-style': ['error', 'unix'],
+    quotes: ['error', 'single'],
+    semi: ['error', 'always'],
+    'no-unused-vars': ['warn'],
+    'no-console': 0,
```

Comment:
Maybe we should check for remaining console.log statements. We don't want to fill up the dev console with useless info when it isn't needed anymore